client/components/RoomList.jsx

```diff
@@ -0,0 +1,52 @@
+RoomList = React.createClass({
+
+  goToRoom(roomId, roomName) {
+    this.props.goToRoom(roomId, roomName);
+  },
+
+  goToNewRoom() {
+    this.props.setView("newRoom");
+  },
+
+  render() {
+    var that = this;
+
+    var loader =  (
+      <div className="ui active inverted dimmer">
+        <div className="ui text loader">
+          Loading messages
+        </div>
+      </div>
+    );
+
+    var joinedRooms = this.props.joinedRooms.map(function(room){
```

Comment:
om du skriver 
`.map(room => {`
istedet for `function`
trenger du ikke lage that = this, siden this fortsatt er bundet til komponenten.